appcenter-analytics/package.json

```diff
@@ -22,14 +22,6 @@
   "dependencies": {
     "appcenter": "2.1.0"
   },
-  "rnpm": {
```

Comment:
Unfortunately we won't be able to completely remove `rnpm` because that would break RN apps on version prior to v0.60.x.